docs/how-to-guides/how-to-create-maps-for-autoware.md

```diff
@@ -0,0 +1,59 @@
+# Map data for Autoware
+
+Autoware relies on high-definition maps (HD maps), including both point cloud and vector maps, to perform various tasks such as localization, route planning, traffic light detection, and other traffic participants' trajectory prediction. In order to use these functionalities in Autoware, compatible HD maps of the driving environment need to be created.
+
+## Creating HD maps
+
+You can use various software, both open-source and proprietary, to create point cloud and vector maps. However, you need to ensure that the output maps are compatible with Autoware. Specifications of point cloud and vector maps that Autoware requires are given below, along with examples of software you can use to create them.
+
+### Point cloud map
+
+A 3D point cloud map is primarily used for LiDAR-based localization in Autoware. In order to determine the current position and orientation of the vehicle, a live scan captured from LiDAR unit(s) is matched against a pre-generated 3D point cloud map. Therefore, an accurate point cloud map is crucial for good localization results. Specifications of an Autoware-compatible point cloud map are as follows:
+
+- A map must cover the entire operational area of the vehicle and should include an additional buffer zone of at least 200 m in all directions.
+- A map is saved using [PCD (Point Cloud Data) file format](https://pointclouds.org/documentation/tutorials/pcd_file_format.html).
+- A map can be saved as a single PCD file or divided into multiple PCD files.
+- Each point in a map must contain X, Y, and Z coordinates.
+- An intensity or RGB value of each point can be included in the PCD file but is optional.
+- A map must be smaller than 1 GB as it is limited by ROS message size.
+- A map resolution should be at least 0.2 m to yield reliable localization results.
+- Although a map can generally be in either local or global coordinates, it must be in global coordinates (georeferenced) if you want to use GNSS data for localization.
+- [Military Grid Reference System (MGRS)](https://en.wikipedia.org/wiki/Military_Grid_Reference_System) is used as a coordinate system for a georeferenced map. In a georeferenced map, the X and Y coordinates of each point represent the point's location within a 100 km MGRS grid, while the Z coordinate represents the point's elevation.
```

Comment:
Should the map follow MGRS when using it for Autoware?

I think this is necessary when the readers want to use MGRS-based GNSS sensor input, but otherwise, IMO the map does not necessarily should follow the MGRS format. (correct me if I am wrong!)

So what do you think about removing this line? 
```suggestion
```

Reply:
Basically, if GNSS is not used it can be in local coordinates as mentioned in the prior bullet point. However, if users want to georeference the map it has to be in MGRS. I think it's important to note that because there are other projections that can be used, UTM, Rectangular plane. Especially, since Autoware.ai used to use Rectangular plane projection and now we switched to MGRS.

Therefore, if you're ok with it I would like to keep this as some users might find it useful.

Reply:
Thanks.

I assumed that we can change the format of map coordinates (among MGRS, UTM, or plane) from [this parameter in `gnss_poser`](https://github.com/autowarefoundation/autoware.universe/blob/f807914b60e70fb1ad4a50bdd942bd2330518fba/sensing/gnss_poser/launch/gnss_poser.launch.xml#L15) when GNSS is available. 
If my understanding is correct, how about something like this?

```suggestion
- When GNSS is available, the coordination system of the map should be compatible with the GNSS data. In the current Autoware Universe, we support MGRS, UTM, and Rectangular plane format for the map (see [the launch file of gnss_pose](https://github.com/autowarefoundation/autoware.universe/blob/main/sensing/gnss_poser/launch/gnss_poser.launch.xml#L15) to set the parameter).
```

Reply:
I see. I think it's reasonable to include other coordinate systems as well. However, it also seems Lanelet2 loader also supports UTM actually (though it's not the [default choice](https://github.com/autowarefoundation/autoware.universe/blob/main/map/map_loader/src/lanelet2_map_loader/lanelet2_map_loader_node.cpp#L53)). 

I will try to reword the sentence a little bit but will reflect your idea of including other coordinate systems. Thank you!